src/src/Helpers/Misc.cpp

```diff
@@ -436,18 +432,22 @@ void HSV2RGBW(float H, float S, float I, int rgbw[4]) {
   const int b = 0;
   rgbw[3] = 255 * (1 - S) * I;
 
-  if (RGB_ORDER == order) {
-    rgbw[0] = r;
-    rgbw[1] = g;
-    rgbw[2] = b;
-  } else if (GBR_ORDER == order) {
-    rgbw[0] = g;
-    rgbw[1] = b;
-    rgbw[2] = r;
-  } else if (BRG_ORDER == order) {
-    rgbw[0] = b;
-    rgbw[1] = r;
-    rgbw[2] = g;
+  switch (order) {
+      case 0: // RGB_ORDER
+          rgbw[0] = r;
+          rgbw[1] = g;
+          rgbw[2] = b;
+          break;
+      case 1: // BRG_ORDER
+          rgbw[0] = b;
+          rgbw[1] = r;
+          rgbw[2] = g;
+          break;
+      case 2: // GBR_ORDER

```

Comment:
As you can see, removing the defines is not making it better as you literally everywhere have to comment what you mean by it.
So the define is better here as it makes the code better readable.